internal/hcsoci/create.go

```diff
@@ -278,6 +280,24 @@ func CreateContainer(ctx context.Context, createOptions *CreateOptions) (_ cow.C
 		if err != nil {
 			return nil, r, err
 		}
+
+		if coi.HostingSystem.OS() == "windows" {
```

Comment:
Should this be behind a feature flag? Maybe users can pass an annotation that enables this for windows containers? 

Are we planning to add support for LCOW too in future?

Reply:
I think it should be fine to automatically do this for hyper-v WCOW, since HCS will do the same for process isolated WCOW (afaik). GCS also creates the mapping for Xenon containers, it's just we don't do it on our end.

This isn't necessary for LCOW, since there's no nested Silo and things "just work" with vsock.